src/pyPROPOSAL/detail/parametrization.cxx

```diff
@@ -144,57 +141,48 @@ void init_parametrization(py::module& m)
             calculation of the countinous randomization.
     )pbdoc";
 
-    auto param_docstring_kinematic_limits = R"pbdoc(
+constexpr auto param_docstring_kinematic_limits = R"pbdoc(
             Get internal hash corresponding to the current parametrization
     )pbdoc";
 
-    py::class_<crosssection::Parametrization<Medium>,
-        std::shared_ptr<crosssection::Parametrization<Medium>>>(
-        m_sub, "ParametrizationForMedium", param_docstring_class)
+template <typename T>
+void decl_param(pybind11::module m_sub, std::string const& class_name)
+{
+    pybind11::class_<crosssection::Parametrization<T>,
```

Comment:
these `pybind11` could all be just `py` to have shorter code

Reply:
:+1: